Modules/_testcapi/pyrun.c

```diff
@@ -0,0 +1,106 @@
+#include "parts.h"
+#include "util.h"
+
+#include <stdio.h>
+#include <errno.h>
+
+char *filename = "pyrun_sample.py";
```

Comment:
Please make this a function scoped variable; if not, you'll need to update the C analyzer script and make an exception for the global variable. (See the failing CI for more info.)

Reply:
Seems that i just foggot to remove it. filename is a parameter from python unit test now